fast64_internal/sm64/sm64_objects.py

```diff
@@ -2277,6 +2283,16 @@ def draw_props(self, layout):
         elif self.export_header_type == "Actor":
             if not self.draw_actor_path(info_box):
                 return
+        elif bpy.context.scene.saveTextures:
+            if self.custom_include_directory:
+                info_box.label(text=f'Include directory "{self.custom_include_directory}"')
+            else:
+                actor_names = self.actor_names
+                joined = ",".join(self.actor_names)
+                if len(actor_names) > 1:
+                    joined = f"{{{joined}}}"
```

Comment:
I find this more readable:
```suggestion
                    joined = "{" f"{joined}" "}"
```